### ISSUE TITLE
Hide model fit progress bar?

### ISSUE THREAD

**Issue by oesteban** [nipreps/eddymotion#181]
@arokem it seems like fitting models now shows a progress bar by default. Is it possible to switch it off?

**Comment by arokem** [comment]
I don't think that you can configure that in DIPY atm, because it's embedded into the [multi-voxel fit decorator](https://github.com/dipy/dipy/blob/master/dipy/reconst/multi_voxel.py#L11). For now, I think that our best bet is something like this: https://stackoverflow.com/a/76946337/3532933 (or one of the other suggestions on that page). I think it could be a configurable input to the "fit" method of these models in a future version, though. I'll post an issue on DIPY (and even try to fix it in my [ongoing saga of a PR](https://github.com/dipy/dipy/pull/2593) there)

**Comment by arokem** [comment]
Something like that implemented in https://github.com/dipy/dipy/pull/2593/commits/44127d54df31516ca453b551dc2728b22380cee4

**Comment by oesteban** [comment]
We also use tqdm. It's going to depend on when tqdm reads the variable

**Comment by arokem** [comment]
Yeah - we should have thought about this (on the DIPY side, I mean). 

Not directly related (might want a separate issue to track execution), but should we implement a "verbose" flag here as well to silence tqdm if-need-be?
